streams/src/test/java/org/apache/kafka/streams/processor/internals/InternalTopicManagerTest.java

```diff
@@ -178,20 +177,6 @@ public void shouldNotThrowExceptionForEmptyTopicMap() {
         internalTopicManager.makeReady(Collections.emptyMap());
     }
 
-    @Test
-    public void shouldExhaustRetriesOnTimeoutExceptionForMakeReady() {
-        mockAdminClient.timeoutNextRequest(1);
-
-        final InternalTopicConfig internalTopicConfig = new RepartitionTopicConfig(topic, Collections.emptyMap());
-        internalTopicConfig.setNumberOfPartitions(1);
-        try {
-            internalTopicManager.makeReady(Collections.singletonMap(topic, internalTopicConfig));
-            fail("Should have thrown StreamsException.");
-        } catch (final StreamsException expected) {
-            assertEquals(TimeoutException.class, expected.getCause().getClass());
-        }
-    }
```

Comment:
If you decide to still log the condition, we could rehabilitate this test to check for the log.